src/Symfony/Component/Console/Input/InputOption.php

```diff
@@ -21,10 +21,25 @@
  */
 class InputOption
 {
+    /**
+     * Do not accept input for this option (e.g. --yell). This is the default behavior of options.
+     */
     public const VALUE_NONE = 1;
+    /**
+     * A value must be passed when the option is used (e.g. --iterations=5 or -i5).
+     */
     public const VALUE_REQUIRED = 2;
+    /**
+     * This option may or may not have a value (e.g. --yell or --yell=loud).
+     */
     public const VALUE_OPTIONAL = 4;
+    /**
+     * This option accepts multiple values (e.g. --dir=/foo --dir=/bar).
+     */
     public const VALUE_IS_ARRAY = 8;
+    /**
+     * The option can be negated (e.g. --ansi or --no-ansi).
```

Comment:
Maybe add that this can only be used combined with `VALUE_NONE`.

Reply:
I tried to apply your suggestion to the doc